druid_setup/single/docker-compose.yml

```diff
@@ -52,7 +51,6 @@ services:
       - ZEN_DRUID_VERSION=${DRUID_VERSION}
 
   postgres:
-    container_name: postgres
```

Comment:
Having a named container can be problematic - when we have multiple "postgres" containers (e.g. one for harmony, and one for druid) - I prefer not naming it unless there's a very good reason for doing so.

Reply:
This comes from the [druid docker-compose](https://github.com/apache/druid/blob/25.0.0/distribution/docker/docker-compose.yml#L33) example. No motivation on our side to have it named, rather just use the default convention.